tests/src/system/basic/WskBasicTests.scala

```diff
@@ -105,6 +105,51 @@ class WskBasicTests
             wsk.pkg.list().stdout should include(name)
     }
 
+    it should "create, and get a package summary" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val packageName = "packageName"
+            val actionName = "actionName"
+            val packageAnnots = Map(
+                "description" -> JsString("packageDesc"),
+                "parameters" -> JsArray(
+                    JsObject(
+                        "name" -> JsString("paramName1"),
+                        "description" -> JsString("paramDesc1")
+                    ),
+                    JsObject(
+                        "name" -> JsString("paramName2"),
+                        "description" -> JsString("paramDesc2")
+                    )
+                )
+            )
+            val actionAnnots = Map(
+                "description" -> JsString("actionDesc"),
+                "parameters" -> JsArray(
+                    JsObject(
+                        "name" -> JsString("paramName1"),
+                        "description" -> JsString("paramDesc1")
+                    ),
+                    JsObject(
+                        "name" -> JsString("paramName2"),
+                        "description" -> JsString("paramDesc2")
+                    )
+                )
+            )
+
+            assetHelper.withCleaner(wsk.pkg, packageName) {
+                (pkg, _) =>
+                    pkg.create(packageName, annotations = packageAnnots)
+            }
+
+            wsk.action.create(packageName + "/" + actionName, defaultAction, annotations = actionAnnots)
+
+            val stdout = wsk.pkg.get(packageName, summary = true).stdout
+            stdout should include regex("""package /.*/packageName: packageDesc\s*\(params: paramName1, paramName2\)""")
```

Comment:
Use variable substitution for the packageName, actionName, and description constant strings defined above.